src/components/TeamMemberCard/index.tsx

```diff
@@ -1,69 +1,31 @@
-import React from "react";
-import {
-  Card,
-  CardContent,
-  Typography,
-  Tooltip,
-  IconButton,
-} from "@mui/material";
+// This code is currently not in use. I am considering if/how this should be structured - right now we have card front, card back, and either the front or back is rendered. This happens withing TeamGridLayout because the state of each card needs to be known by the other cards to manage flipping to the back/front when another card is clicked. I would prefer to have TeamGridLayout contain only TeamMemberCard, and TeamMemberCard hold the logic of back/front - but currently do not know how to do that. (maybe the state can be passed as a prop from teamgridlayout down to teammembercard?)
```

Comment:
These explanatory comments are great. I appreciate them lots. 

Many folks want to eliminate anything like this, thinking that checked-on code should admit nothing unfinished, no loose ends, or it would be EMBARASSING. 

But please, how often is code set-in-stone DONE? Much better to acknowledge the areas of potential improvement, or uncertainty, or whatever, that will set up the next person (whether you or someone else) to carry on with the code.